main/util/src/mill/util/Jvm.scala

```diff
@@ -0,0 +1,34 @@
+package mill.util
+
+import mill.api.Loose.Agg
+object Jvm {
```

Comment:
I wonder why we don't move the whole `Jvm` object to `mill.util`, as it is a dependency of `mill.core`. When we make it a trait first, we can still inherit it in both `Jvm` objects for backwards compatibility.

Reply:
That's a good question.

The thing is I don't really know what should go in `mill.modules.*`. It seems like a "bunch of utils code for users to use instead of us" kind of thing. Not sure if that's what it should be, but that's what it kind of is right now.

For now, resolving the question of what `mill.modules.*` should be is out of scope of this PR, so I only moved what i needed and left a forwarder to avoid affecting any downstream code.

Reply:
ok.